modules/ROOT/pages/troubleshoot-agent-network.adoc

```diff
@@ -0,0 +1,41 @@
+= Troubleshooting Your Agent Network
+
+Use Agent Visualizer with Anypoint Monitoring to diagnose issues across agents and MCP servers. Open a node’s detail card to see associated metrics and access logs and traces in Anypoint Monitoring.
+
+[[view-metrics]]
+== View Metrics
+
+. Select a node to open the detail card.
+. In *Performance and Health*, click *View metrics* to view additional metrics in Anypoint Monitoring Insights. 
+
+For more information, see xref:monitoring::anypoint-insights-agentic.adoc[].
+
+[[access-logs]]
+== Access Logs
+
+On US and EU Clouds, Anypoint Monitoring aggregates log files so you can manage, search for, filter, and analyze your logs. You can use aggregated logs in conjunction with monitoring to help identify issues and investigate failures quickly. 
```

Comment:
I wonder if we should just focus on the idea that they can get logs for the instance. I'm assuming the link will go directly to those logs, and they won't have to manage, search for, filter, those logs, just analyze. I mean, they could change whatever search shows up.

To verify what I'm saying, I'll try to remember to ask Sathish what search we're using.